src/components/Aqidata.jsx

```diff
@@ -0,0 +1,135 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation } from 'react-router-dom';
+import millify from 'millify';
+import Header from './Header';
+import { displayWeatherData } from '../redux/Aqidata/aqidataSlice';
+
+const Aqiweather = () => {
+  const location = useLocation();
+  const { city } = location.state;
+  const { weather } = useSelector((state) => state.weather);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const obj = {
+      lat: city.coordinates[0],
+      lon: city.coordinates[1],
+    };
+    dispatch(displayWeatherData(obj));
+  }, [city.coordinates, dispatch]);
+
+  const weatherList = weather?.list || [];
+
+  return (
+    <div className="bg-[#e94988]">
+      <Header title={city.cityName[0]} />
+      <table className="table-fixed bg-[#e94988] w-full mb-4">
+        <thead>
+          <tr className="bg-[#e94988] text-white">
+            <td>
+              <div className="flex justify-between items-start">
+                <div className="px-3 py-4 items-start flex cursor-pointer">
+                  <img src={city.flag} alt={city.alt} width="200" />
+                </div>
+                <div className="px-3 py-4 flex flex-col items-end cursor-pointer">
+                  <h2 className="font-semibold">Country</h2>
+                  <p>{city.country}</p>
+                  <h2 className="font-semibold">Population</h2>
+                  <p>{millify(city.population)}</p>
+                </div>
+              </div>
+            </td>
+          </tr>
+          <tr>
+            <th>
+              <div className="flex justify-center items-center">
+                <div className="px-3 py-4 items-center flex cursor-pointer">
+                  <h4 className="text-white">Weather Data</h4>
+                </div>
+              </div>
+            </th>
+          </tr>
+        </thead>
+        {weatherList.map((weatherData) => (
+          <tbody key={weatherData.dt}>
+            <tr className="bg-[#df4782] text-white">
+              <td className="w-1/6 whitespace-wrap px-2 py-2">
+                <div className="flex items-start justify-between">
+                  <h2 className="font-bold">AQI</h2>
+                  <h3>{weatherData.main.aqi}</h3>
+                </div>
+              </td>
+            </tr>
+            <tr className="bg-[#e94988] text-white">
+              <td className="w-1/6 whitespace-wrap px-2 py-2">
+                <div className="flex items-start justify-between">
+                  <h2 className="font-bold">CO</h2>
+                  <h3>{weatherData.components.co}</h3>
+                </div>
+              </td>
+            </tr>
```

Comment:
- [ ] Looking at your cards you would notice that you have the details of each card displayed in its own container as compared to that of the given design. In your card, you have `country`, `city`, `population`, and `icon` displayed in a `row` when they should be displayed in a `column`.

- [ ] Kindly make sure that your card `icon` is displayed absolutely at the top of your card to match that of the given design. To have a more visual understanding of the changes kindly take a look at the screenshot below for more details:

<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="115" alt="Screenshot 2023-08-25 at 09 17 19" src="https://github.com/Kanyorok/air-pollution-aqi/assets/107192076/0503ada6-3977-41b6-9e8b-e6e617ce7370">
</td>
<td>
<img width="339" alt="Screenshot 2023-08-25 at 09 16 43" src="https://github.com/Kanyorok/air-pollution-aqi/assets/107192076/621179a6-32bc-40f1-9851-8ffa2ace2818">
</td>
</tr>
</table>